manifests/plugin/dhcp/infoblox.pp

```diff
@@ -0,0 +1,32 @@
+# = Foreman Proxy Infoblox DHCP
+#
+# This class installs the Infoblox DNS plugin for Foreman proxy
+#
+# === Parameters:
+#
+# $username::    The username of the Infoblox user
+#
+# $password::    The password of the Infoblox user
+#
+# $record_type:: Record type to manage, can be "host" or "fixedaddress"
+#
+# $use_ranges::  Use  pre-definded ranges in networks to find available IP's
+#                type:boolean
+#
+class foreman_proxy::plugin::dhcp::infoblox (
+  $username    = $::foreman_proxy::plugin::dhcp::infoblox::params::username,
+  $password    = $::foreman_proxy::plugin::dhcp::infoblox::params::password,
+  $record_type = $::foreman_proxy::plugin::dhcp::infoblox::params::record_type,
+  $use_ranges  = $::foreman_proxy::plugin::dhcp::infoblox::params::use_ranges,
+) inherits foreman_proxy::plugin::dhcp::infoblox::params {
+  validate_string($username, $password)
```

Comment:
Then thing is, I want to force user to specify the values, having default for user credentials doesn't make much sense to me. What I think is happening is kafo validating values also for non-enabled plugins, I will try to fix this in kafo, and based on how hard the fix is, I will either do it in kafo or fix it here temporary